### PR TITLE
Add new command to generate board page for a specific Filament resource

### PR DESCRIPTION
**Pull request by tgeorgel**
- Adapted the KanbanBoard class so one can extend the `Filament\Resources\Pages\Page` instead of `Filament\Pages\Page`
- Added a command and a stub to use this now class instead of the default page class

### CODE REVIEW

**Comment by mokhosh** [comment]
Thanks for making this PR.

However:

1. You have errors in the code, `use`ing the same class twice in the service provider
2. All tests are failing
3. You shouldn't make a PR from your `main` to my `main`. Make a PR from a feature branch to my `2.x`
4. There are unrelated changes in the PR
5. It would be good if you could add some tests

Once you do these I can begin to pull your PR and test it and review it.

**Comment by tgeorgel** [comment]
Hi @mokhosh 

Thanks for your feedback. I'll fix those issues. Can you just explain what your mean by unrelated changes ?

Thanks

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ php artisan filament-kanban:install
 
 ## Before You Start
 
-> [!IMPORTANT]
```
Comment (mokhosh) [inline README.md:33]: This line for example doesn't have to change for this PR.

File: README.md [hunk 2]
```diff
@@ -270,7 +291,7 @@ Are you a visual learner? I have created some Youtube videos to get you started
 > [!WARNING]
 > These videos are recorded with version 1.x of the package.
 > It is now much simpler to use the package, and requires much less code from you.
-> 
+>
```
Comment (mokhosh) [inline README.md:294]: Or this one.

File: src/FilamentKanbanServiceProvider.php [hunk 3]
```diff
@@ -2,16 +2,17 @@
 
 namespace Mokhosh\FilamentKanban;
 
-use Filament\Support\Assets\Asset;
```
Comment (mokhosh) [inline src/FilamentKanbanServiceProvider.php:5]: Or the order of imports here.

File: README.md (hunk 1 above)
Comment (tgeorgel) [reply README.md:33]: > This line for example doesn't have to change for this PR.

Oh, I understand now. However, it's my code editor that automatically trims additional spaces. If that's a problem I'll try disabling it for the MD file edition.

**Pull request closed without merging**